pyriemann/datasets/sampling.py

```diff
@@ -324,6 +475,13 @@ def sample_gaussian_spd(n_matrices, mean, sigma, random_state=None, n_jobs=1):
     n_jobs : int, default=1
         The number of jobs to use for the computation. This works by computing
         each of the class centroid in parallel. If -1 all CPUs are used.
+    sampling_method : str, default='auto'
+        Name of the sampling method used to sample samples_r. It can be
```

Comment:
Not sure that `samples_r` is understandable by any user reading the doc.
```suggestion
        Sampling method to sample eigenvalues. It can be
```